.github/workflows/linters.yml

```diff
@@ -0,0 +1,30 @@
+name: Run Linters
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+
+jobs:
+  linters-check:
+    name: Run Linters
+    runs-on: ubuntu-20.04
+
+    steps:
+      - name: Check out Git repository
+        uses: actions/checkout@v3
+
+      - name: Set up Python
+        uses: actions/setup-python@v3
+        with:
+          python-version: 3.9.10
+
+      - name: Install Python dependencies
+        run: pip install flake8 mypy black
+
+      - name: Run flake8
+        uses: wearerequired/lint-action@v1
+        with:
+          flake8: true
+          black: true
+          mypy: true
```

Comment:
Do we need mypy? I think he can get in a lot of trouble.

Reply:
Ok, will remove it

Reply:
Done